wandb/cli/cli.py

```diff
@@ -886,7 +938,8 @@ def _parse_settings(settings):
                 "queue": queue,
                 "run_spec": json.dumps(
                     construct_launch_spec(
-                        os.getcwd(),  # uri,
+                        None, # uri
```

Comment:
@KyleGoyette , does this util encompass all that is needed to create a launch spec from a job or should Hugo use something else?

Also, is the user responsible for creating the launch spec or does the launch API just take the job argument nowadays?

Reply:
The launch api just takes the job argument. And the constructed launch spec should work, it does all the required merging.